native-api.md

```diff
@@ -0,0 +1,39 @@
+API for native mobile frontends
+-------------------------------
+**Retrieve fixed data for all Course Journal XBlocks in a course:**
+```
+GET https://<lms_server_url>/api/courses/v1/blocks/?course_id=<course_id>&username=<username>&depth=all&requested_fields=student_view_data
+```
+
+Example return value:
+```
+"student_view_data": {
+    "display_name": "Course Journal",
+    "answer_sections": [{
+        'name': u'First Section', 
+        'questions': [{
+            'answer': 'Not answered yet.', 
+            'question': u'Hello, who are you? What\u2019s your name?'
+        }, {
+            'answer': 'student input', 
+            'question': 'Tell us more about yourself.'
+            }
+        ]}
+    ],
+    "display_answers": true,
+    "display_user_metrics": true,
+    "progress": {
+        "user": 22,
+        "cohort_average": 67"
+    },
+    "proficiency": {
+        "user": 44,
+        "cohort_average": 32
+    },
+    "display_key_takeaways": true,
+    "key_takeaways_pdf_url": "/path/to/takeaways.pdf",
+    "pdf_report_url": "/path/to/report.pdf",
+    "pdf_report_link_heading": "PDF Report",
+    "pdf_report_link_text": "PDF Report Link"
+},
```

Comment:
As mentioned previously, the user-specific data should be returned by the `student_view_user_state` endpoint.  See [Problem Builder docs](https://github.com/open-craft/problem-builder/blob/master/doc/Native%20APIs.md#native-api-documentation) for how that endpoint URL is constructed.

Reply:
While the PDF contents are different for all users, the PDF link is the same. So in this case where should that data belong?

Reply:
Hmm...  borderline call there..  I think it's ok to keep that in the `student_view_data` then, but maybe make a note in the docs to call it out as special?